book/src/background/fields.md

```diff
@@ -193,6 +193,18 @@ of $\mathbb{F}_p^\times$ must divide $p-1.$
 
 [lagrange-group]: https://en.wikipedia.org/wiki/Lagrange%27s_theorem_(group_theory)
 
+[PLONK-based] proving systems like Halo 2 are more convenient to use with fields that have
+a large number of multiplicative subgroups with a "smooth" distribution (which makes the
+performance cliffs smaller and more granular as circuit sizes increase). The Pallas and
```

Comment:
That's true, but also specifically 2-adic subgroups are nice for efficient FFTs.